app/backend/src/core/clients/coingecko/assets/addL2Addresses.ts

```diff
@@ -0,0 +1,152 @@
+import fs from "fs/promises";
+import path from "path";
+import axios from "axios";
+import { WhitelistedAsset } from "@/core/clients/coingecko/assets/types";
+import { Logger } from "@/tools/Logger";
+
+const ChainIds = { Ethereum: 1, Optimism: 10, Starknet: -1 };
```

Comment:
Not sure what `chainId` to use for Starknet.
According to this [this](https://stackoverflow.com/a/72909465) it's either `'SN_MAIN'` or `23448594291968334`. First one is a string and second values doesn't properly fit into `number`. Should we switch to `BigInt`?